### PR TITLE
Lot's of Little Changes and Big Fixes

### PR DESCRIPTION
**Pull request by Tresky**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ app.use((req, res, next) => {
   // Request methods you wish to allow
   res.setHeader('Access-Control-Allow-Methods', 'GET, POST, OPTIONS, PUT, PATCH, DELETE')
   // Request headers you wish to allow
-  res.setHeader('Access-Control-Allow-Headers', 'X-Requested-With,content-type,authorization')
+  res.setHeader('Access-Control-Allow-Headers', 'X-Requested-With,Content-Type,Authorization')
```
Comment (TylerGauntlett) [inline src/app.js:38]: Space between , and Authorization

**Review by TylerGauntlett — CHANGES_REQUESTED** [review]

File: src/app.js (hunk 1 above)
Comment (Tresky) [reply src/app.js:38]: It actually shouldn't be there. It's not an array. It's a CSV string, so spaces are usually not placed.

**Review by TylerGauntlett — APPROVED** [review]

File: src/app.js (hunk 1 above)
Comment (TylerGauntlett) [reply src/app.js:38]: Good enough for me

**Comment by TylerGauntlett** [comment]
DOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOITDOIT

**Comment by TylerGauntlett** [comment]
AHHHHHHHHHHHH

**Pull request merged**